Apps/Contoso.WPF.Puppet/MainWindow.xaml

```diff
@@ -10,7 +10,7 @@
         mc:Ignorable="d"
         Title="App Center WPF Puppet App" Height="505" Width="335">
     <StackPanel>
-        <TabControl SelectionChanged="TabControl_SelectionChanged" Height="185">
+        <TabControl SelectionChanged="TabControl_SelectionChanged" Height="474">
```

Comment:
Is it related?

Reply:
I think that's unintentional change caused by IDE automatically